package.json

```diff
@@ -6,150 +6,153 @@
     "node": ">=6",
     "npm": ">=3.8"
   },
+  "scripts": {
+    "start": "node tools/run.js",
+    "build": "node tools/build.js",
+    "build:debug": "node tools/build.js --debug",
+    "build:lib": "node tools/lib.build.js",
+    "test": "jest --config jest.config.json",
+    "test:coverage": "jest --config jest.config.json --coverage",
+    "test:watch": "jest --config jest.config.json --watchAll",
+    "test:update": "jest --config jest.config.json -u",
+    "lint": "yarn run eslint && yarn run stylelint",
+    "lint:fix": "yarn run eslint --fix && stylefmt -r \"src/**/*.css\"",
+    "eslint": "eslint .",
+    "stylelint": "stylelint --config stylelint.config.js \"src/**/*.{js,css}\"",
+    "flow:check-all": "flow check",
+    "flow:install-types": "flow-typed install --ignoreDeps dev",
+    "locales:missing": "node tools/findMissingLocales"
+  },
   "dependencies": {
-    "@babel/polyfill": "~7.0.0-beta.44",
+    "@babel/polyfill": "^7.0.0-beta.54",
     "babel-plugin-transform-builtin-extend": "^1.1.2",
-    "classnames": "~2.2.5",
-    "detect-browser": "^2.2.0",
+    "classnames": "^2.2.6",
+    "detect-browser": "^3.0.0",
     "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-node": "^6.0.1",
+    "eslint-plugin-node": "^7.0.1",
     "eslint-plugin-promise": "^3.8.0",
-    "font-awesome": "~4.7.0",
-    "history": "~4.7.2",
-    "i18next": "~11.1.1",
-    "i18next-browser-languagedetector": "~2.2.0",
-    "lodash": "~4.17.5",
-    "moment": "^2.22.0",
-    "moment-timezone": "^0.5.14",
-    "normalize-url": "^1.9.1",
-    "normalize.css": "~8.0.0",
-    "react": "~16.3.1",
-    "react-dom": "~16.3.1",
-    "react-fontawesome": "~1.6.1",
+    "font-awesome": "^4.7.0",
+    "history": "^4.7.2",
+    "i18next": "^11.3.6",
+    "i18next-browser-languagedetector": "^2.2.2",
+    "lodash": "^4.17.10",
+    "moment": "^2.22.2",
+    "moment-timezone": "^0.5.21",
+    "normalize.css": "^8.0.0",
```

Comment:
Don't we need normalize-url anymore ? (:tada: :pray: )

Reply:
Nope. Found normalize-path because newer versions of normalize-url actually work only with urls :+1: